tests/core_test.py

```diff
@@ -69,6 +69,12 @@ def f(*args):
     assert False
   return f
 
+def get_jaxpr_eqn(jaxpr, prim_name):
```

Comment:
You could replace the call sites of this function with the one-liner `next(eqn for eqn in jaxpr.eqns if eqn.primitive.name == 'cond')` or similar.